src/numeric_filter.c

```diff
@@ -25,7 +26,7 @@ int parseDoubleRange(const char *s, double *target, int isMin,
   }
   char *endptr = NULL;
   errno = 0;
-  *target = strtod(s, &endptr);
+  *target = fast_float_strtod(s, &endptr);
```

Comment:
Not originating in this PR, but don't we also need to check `errno`, e.g., like in `ParseDouble` where it can also be `EINVAL` (not only `ERANGE` which will set target to +-`HUGE_VAL`)